src/sidebar/components/MarkdownView.tsx

```diff
@@ -50,14 +108,63 @@ export default function MarkdownView({
   // a review in the future.
   return (
     <div className="w-full break-anywhere cursor-text">
-      <StyledText>
+      <StyledText
+        classes={classnames({
+          // A `relative` wrapper around the `Popover` component is needed for
+          // when the native Popover API is not supported.
+          relative: mentionsEnabled,
+        })}
+      >
         <div
           className={classes}
           data-testid="markdown-text"
           ref={content}
           dangerouslySetInnerHTML={{ __html: html }}
           style={style}
+          // React types do not define `onMouseEnterCapture`, but preact does
+          // eslint-disable-next-line react/no-unknown-property
```

Comment:
Disabling `react/no-unknown-property` here is needed because it evaluates props based on react types, not preact ones, where the `Capture` variants of event handlers are not defined.

We discussed [in slack](https://hypothes-is.slack.com/archives/C1M8NH76X/p1739358787349359?thread_ts=1739357853.490339&cid=C1M8NH76X) the possibility of completely disabling `react/no-unknown-property`, since TypeScript covers the same more accurately, but I will do that on a PR of its own.